presentation.py

```diff
@@ -53,36 +48,34 @@ def add_side_title(slide, title):
     p.alignment = PP_ALIGN.LEFT
     p.text = title
 
-def add_img(slide, img_file):
+def add_img(slide, img_file, direction = Direction.VERTICAL):
     img = Image.open(img_file)
     w, h = img.size
     ratio = h / w
-    max_width = SLIDE_WIDTH - SIDE_MARGIN * 2
-    max_height = SLIDE_HEIGHT - TITLE_HEIGHT - TOP_MARGIN * 2
-    if max_height / max_width > ratio:
-        width = max_width
-        height = max_width * ratio
-    else:
-        height = max_height
-        width = height / ratio
-    left = (SLIDE_WIDTH - width) / 2
-    top = SLIDE_HEIGHT - height - TOP_MARGIN
-    slide.shapes.add_picture(img_file, left, top, width=width)
 
-def add_side_img(slide, img_file):
-    img = Image.open(img_file)
-    w, h = img.size
-    ratio = h / w
-    max_height = SLIDE_HEIGHT  - TOP_MARGIN * 2
-    max_width = SLIDE_WIDTH / 2 - SIDE_MARGIN
-    if max_height / max_width > ratio:
-        width = max_width
-        height = max_width * ratio
+    if direction == Direction.HORIZONTAL:
+        max_width = SLIDE_WIDTH - SIDE_MARGIN * 2
+        max_height = SLIDE_HEIGHT - TITLE_HEIGHT - TOP_MARGIN * 2
+        if max_height / max_width > ratio:
```

Comment:
There are two completely same if-then statements. They can be refactored (and I know that my code can be refactored a lot, too).